### ISSUE TITLE
[libtasn1] build failure in osx pipeline

### ISSUE THREAD

**Issue by autoantwort** [microsoft/vcpkg#17098]
**Host Environment**
- OS: macOS CI
- Compiler: unknown

**To Reproduce**
Run pipeline of #17073

**Failure logs** 
https://dev.azure.com/vcpkg/c1ee48cb-0df2-4ab3-8384-b1df5a79fe53/_apis/build/builds/51525/artifacts?artifactName=failure%20logs%20for%20x64-osx&api-version=6.0&%24format=zip

autoconf-x64-osx-err.log:
```
autoreconf: export WARNINGS=
autoreconf: Entering directory '.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal --force -I m4 -I m4-gl
autoreconf: configure.ac: tracing
autoreconf: running: glibtoolize --copy --force
autoreconf: configure.ac: not using Intltool
autoreconf: running: gtkdocize --copy
Can't exec "gtkdocize": No such file or directory at /usr/local/Cellar/autoconf/2.71/share/autoconf/Autom4te/FileUtils.pm line 293.
autoreconf: error: gtkdocize failed with exit status: 2
```

**Additional context**
It seems to be similar to #16990

**Comment by autoantwort** [comment]
Seems to be the same error as #17057

**Comment by NancyLi1013** [comment]
Hi @autoantwort 

Thanks for posting this issue.
Sorry, I cannot repro this on my machine.

```
vcpkg@VCPKGMM-12 vcpkg % ./vcpkg install libtasn1 --editable
Computing installation plan...
The following packages will be built and installed:
    libtasn1[core]:x64-osx -> 4.16.0
Could not locate cached archive: /Users/vcpkg/.cache/vcpkg/archives/.zip
Starting package 1/1: libtasn1:x64-osx
Building package libtasn1[core]:x64-osx...
-- Downloading https://ftp.gnu.org/gnu/libtasn1/libtasn1-4.16.0.tar.gz -> libtasn1-4.16.0.tar.gz...
-- Extracting source /Users/vcpkg/Nancy/vcpkg/downloads/libtasn1-4.16.0.tar.gz
-- Using source at /Users/vcpkg/Nancy/vcpkg/buildtrees/libtasn1/src/4.16.0-1a71df00ff
-- Configuring x64-osx-dbg
-- Configuring x64-osx-rel
-- Generating configure for x64-osx
-- Finished generating configure for x64-osx
-- Configuring x64-osx-dbg
-- Configuring x64-osx-rel
-- Building x64-osx-dbg
-- Installing x64-osx-dbg
-- Building x64-osx-rel
-- Installing x64-osx-rel
-- Fixing pkgconfig file: /Users/vcpkg/Nancy/vcpkg/packages/libtasn1_x64-osx/lib/pkgconfig/libtasn1.pc
-- Fixing pkgconfig file: /Users/vcpkg/Nancy/vcpkg/packages/libtasn1_x64-osx/debug/lib/pkgconfig/libtasn1.pc
-- Installing: /Users/vcpkg/Nancy/vcpkg/packages/libtasn1_x64-osx/share/libtasn1/copyright
-- Performing post-build validation
-- Performing post-build validation done
Building package libtasn1[core]:x64-osx... done
Installing package libtasn1[core]:x64-osx...
Installing package libtasn1[core]:x64-osx... done
Elapsed time for package libtasn1:x64-osx: 56.01 s

Total elapsed time: 56.01 s
```
Could you please help confirm if this issue exists on your local machine?

**Comment by autoantwort** [comment]
Yes I also have this error locally

**Comment by autoantwort** [comment]
`brew install gtk-doc` solves the issue

**Comment by dg0yt** [comment]
I think this is now fixed by #17420.

**Comment by NancyLi1013** [comment]
Thanks for your kindly reminder. @dg0yt
